tests/Controller/AbstractControllerTest.php

```diff
@@ -10,7 +11,7 @@
 abstract class AbstractControllerTest extends WebTestCase
 {
     use Factories;
-    protected static TranslatorInterface $translator;
+    protected static object $translator;
```

Comment:
On peut pas faire mieux ici ? Avec une union de types par ex >

Reply:
`object|TranslatorInterface` -> type is dupplicated by object